css/index.css

```diff
@@ -140,7 +141,313 @@ footer nav {
+    width: 12.5%;
+    height: 100px;
+    color: white;
+    margin: 20px 2.5%;
+    background: forestgreen;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box6:hover {
+    width: 12.5%;
+    height: 100px;
+    color: forestgreen;
+    margin: 20px 2.5%;
+    background: black;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box7 {
+    width: 12.5%;
+    height: 100px;
+    color: white;
+    margin: 20px 2.5%;
+    background: darkorchid;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box7:hover {
+    width: 12.5%;
+    height: 100px;
+    color: darkorchid;
+    margin: 20px 2.5%;
+    background: black;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box8 {
+    width: 12.5%;
+    height: 100px;
+    background: white;
+    margin: 20px 2.5%;
+    background: hotpink;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box8:hover {
+    width: 12.5%;
+    height: 100px;
+    background: hotpink;
+    margin: 20px 2.5%;
+    background: black;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.middle-content .boxes .box9 {
+    width: 12.5%;
+    height: 100px;
+    color: white;
+    margin: 20px 2.5%;
+    background: indigo;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.middle-content .boxes .box9:hover {
+    width: 12.5%;
+    height: 100px;
+    color: indigo;
+    margin: 20px 2.5%;
+    background: black;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.middle-content .boxes .box10 {
+    width: 12.5%;
+    height: 100px;
+    background: dodgerblue;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.middle-content .boxes .box10:hover {
+    width: 12.5%;
+    height: 100px;
+    background: black;
+    margin: 20px 2.5%;
+    color: dodgerblue;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.section {
+    display: flex; 
+    flex-wrap: wrap;
+    justify-content: space-between;
+    /* trbl */
+    margin: 10px 10px 10px 10px
+}
+
+.wrapper2 {
+    border-top: 2px solid black;
+    border-style:dotted;
+	width: 120%;
+	/* TRBL */
+	margin: 10px 0 0 0px;
+}
+
+.together {
+    /* TRBL */
+    margin: 20px 0 20px 0;
+}
+
+.strategy {
+    width: 300px;
+    height: 400px;
+    border: 1 px solid red;
+    
+}
+
+button {
+    background: white;
+    color: black;
+    height: 40px;
+    width: 120px;
+    border:1px solid black;
+    font-size: 14px;
+    padding: 10px;
+    margin: 10px 0 10px 0;
+}
+
+button:hover {
+    background: black;
+    color: white;
+}
+
+.header-logo {
+    display:flex;
+    justify-content: flex-start;
+}
+
```

Comment:
Here is the correct code to make it match perfectly
`html, body, div, span, applet, object, iframe,
h1, h2, h3, h4, h5, h6, p, blockquote, pre,
a, abbr, acronym, address, big, cite, code,
del, dfn, em, img, ins, kbd, q, s, samp,
small, strike, strong, sub, sup, tt, var,
b, u, i, center,
dl, dt, dd, ol, ul, li,
fieldset, form, label, legend,
table, caption, tbody, tfoot, thead, tr, th, td,
article, aside, canvas, details, embed, 
figure, figcaption, footer, header, hgroup, 
menu, nav, output, ruby, section, summary,
time, mark, audio, video {
	margin: 0;
	padding: 0;
	border: 0;
	font-size: 100%;
	font: inherit;
	vertical-align: baseline;
}
/* HTML5 display-role reset for older browsers */
article, aside, details, figcaption, figure, 
footer, header, hgroup, menu, nav, section {
	display: block;
}
body {
	line-height: 1;
}
ol, ul {
	list-style: none;
}
blockquote, q {
	quotes: none;
}
blockquote:before, blockquote:after,
q:before, q:after {
	content: '';
	content: none;
}
table {
	border-collapse: collapse;
	border-spacing: 0;
}

* {
    box-sizing: border-box;
}

html, body {
    height: 100%;
    font-family: 'Roboto', sans-serif;
}

h1, h2, h3, h4, h5 {
    font-size: 18px;
    margin-bottom: 15px;
    font-family: 'Rubik', sans-serif;
}

p {
    line-height: 1.4;
}

.container {
    width: 800px;
    margin: 0 auto;
}

header {
    width: 100%;
    display: flex;
    margin: 20px 0;
}

header .logo {
    width: 157px;
    height: 40px;
    background: black;
}

header nav {
    width: 100%;
    display: flex;
    justify-content: space-around;
    align-items: flex-end;
}

header nav a {
    font-size: 14px;
    text-decoration: none;
    color: black;
}

.jumbotron {
    width: 100%;
    height: auto;
    margin: 20px 0;
}

.top-content {
    display: flex;
    flex-wrap: wrap;
    justify-content: space-evenly;
    margin-bottom: 20px;
    border-bottom: 1px dashed black;
}

.top-content .text-container {
    width: 48%;
    padding: 0 1%;
    padding-bottom: 20px;  
}

.middle-content {
    margin-bottom: 20px;
    border-bottom: 1px dashed black;
}

.middle-content h2 {
    padding: 0 2%;
    margin-bottom: 0;
}

.middle-content .boxes {
    display: flex;
    flex-wrap: wrap;
    justify-content: space-evenly;
}

.middle-content .boxes .box {
    width: 12.5%;
    height: 100px;
    background: black;
    margin: 20px 2.5%;
    color: white;
    display: flex;
    align-items: center;
    justify-content: center;
}

.middle-content .boxes .box1 {
  background: teal;   
}

.middle-content .boxes .box2 {
  background: gold;   
}

.middle-content .boxes .box3 {
  background: cadetblue;   
}

.middle-content .boxes .box4 {
  background: coral;   
}

.middle-content .boxes .box5 {
  background: crimson;   
}

.middle-content .boxes .box6 {
  background: forestgreen;   
}

.middle-content .boxes .box7 {
  background: darkorchid;   
}

.middle-content .boxes .box8 {
  background: hotpink;   
}

.middle-content .boxes .box9 {
  background: indigo;   
}

.middle-content .boxes .box10 {
  background: dodgerblue;   
}

.bottom-content {
    display: flex;
    margin: 0 2% 20px;
    justify-content: space-around;
}

.bottom-content .text-container {
    padding-right: 4%;
}

.bottom-content .text-container:last-child {
    padding-right: 0;
}

.about-page .about-top {
 display: flex;
}

.about-page .about-text {
  width: 350px;
}
.about-page .about-intro {
  padding: 0 20px 20px;
  border-bottom: 1px dashed black;
}

.about-page .about-content {
  display: flex;
  flex-wrap: wrap;
}

.about-page .about-content .about-box {
  width: 370px;
  margin: 30px 15px;
  border-bottom: 1px dashed black;

}

.about-page .about-content .about-box h2 {
  margin-top: 10px;
}

.about-page .about-content .about-box .about-btn {
  border: 2px solid black;
  border-radius: 10px;
  padding: 10px;
  width: 130px;
  text-align: center;
  cursor: pointer;
  margin: 20px 0;
}

.about-page .about-content .about-box .about-btn:hover {
  background: black;
  color: white;
}


.about-page .about-footer-content {
  margin-bottom: 20px;
  padding: 20px;
}

footer {
    width: 100%;
    background: black;
}

footer nav {
    width: 60%;
    display: flex;
    justify-content: space-between;
    align-items: center;
    padding: 20px 2%;
    font-size: 14px;
}

footer nav a {
    color: white;
    text-decoration: none;
}`